glide.lock

```diff
@@ -80,12 +97,14 @@ imports:
 - name: github.com/spf13/viper
   version: 25b30aa063fc18e48662b86996252eabdcf2f0c7
 - name: github.com/stretchr/testify
-  version: 890a5c3458b43e6104ff5da8dfa139d013d77544
+  version: 69483b4bd14f5845b5a1e55bca19e954e827f1d0
   subpackages:
   - assert
   - require
+- name: github.com/whyrusleeping/go-logging
```

Comment:
Yet another logging library number 2.

Reply:
Yeah these also had me doubting this library. Do we think it's worth implementing and maintaining the socket logic ourselves? I'd also saw if we do so we'd drop any kind of portability (while this option is disabled).

Reply:
I'm leaning towards just implementing it ourselves, or checking if there's any other library with a reduced footprint.  On the portability issue, abstraction layers are not hard to create.

Reply:
Yeah but I don't want to own the complexities of maintaining socket code for one platform, let alone multiple.